File: UNXNavigatorDemo/Feature/Basic/Controllers/ViewController03_Transparent.m
//
//  ViewController03_Transparent.m
//  UNXNavigatorDemo
//
//  Created by Leo Lee on 2020/10/26.
//

#import <UNXNavigator/UNXNavigator.h>

#import "ViewController03_Transparent.h"

@interface ViewController03_Transparent ()

@end

@implementation ViewController03_Transparent

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (UIColor *)unx_navigationBarBackgroundColor {
    return [UIColor clearColor];
}

- (UIColor *)unx_shadowImageTintColor {
    return [UIColor clearColor];
}

@end

File: Example/UNXNavigatorSwiftDemo/Basic/ViewController03_Transparent.swift
//
//  ViewController03_Transparent.swift
//  UNXNavigatorSwiftDemo
//
//  Created by Leo Lee on 2020/11/23.
//

import UIKit

class ViewController03_Transparent: BaseTableViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }
    
    override var unx_navigationBarBackgroundColor: UIColor? {
        return .clear
    }
    
    override var unx_shadowImageTintColor: UIColor? {
        return .clear
    }

}
